src/NUnitFramework/framework/Internal/Commands/MaxTimeCommand.cs

```diff
@@ -62,12 +62,11 @@ public override TestResult Execute(TestExecutionContext context)
             TestResult testResult = innerCommand.Execute(context);
 
             long tickCount = Stopwatch.GetTimestamp() - startTicks;
-            double seconds = (double)tickCount / Stopwatch.Frequency;
-            testResult.Duration = TimeSpan.FromSeconds(seconds);
+            testResult.Duration = (double)tickCount / Stopwatch.Frequency;
```

Comment:
This should now be in seconds. Maybe a comment would help either here or below, where you convert to ms. When we used FromSeconds and TotalMilliseconds it was self-explanatory but I had to read this a few times to catch on.